plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -520,12 +522,160 @@ int add_event(struct flb_cloudwatch *ctx, struct cw_flush *buf,
     return 0;
 }
 
+int should_add_to_emf(struct flb_intermediate_metric *an_item)
+{
+    /* Valid for cpu plugin */
+    if (strncmp(an_item->key.via.str.ptr, "cpu_p", 4) == 0 
```

Comment:
```cpu_p``` string length is 5, not 4.

Reply:
I think `strncmp` compares the first n chars regardless the length of the string. I updated it.